app/views/claims/schools/claims/mentor_trainings/edit.html.erb

```diff
@@ -0,0 +1,59 @@
+<% content_for(
+  :page_title,
+  t(".page_title", mentor: mentor_training_form.mentor_training.mentor_full_name),
+) %>
+
+<% render "claims/schools/primary_navigation", school: @school, current: :claims %>
+
+<%= content_for(:before_content) do %>
+  <%= govuk_back_link(href: mentor_training_form.back_path) %>
+<% end %>
+
+<div class="govuk-width-container">
+  <%= form_with(
+    model: mentor_training_form,
+    url: claims_school_claim_mentor_training_path(
+      @school,
+      mentor_training_form.claim,
+      mentor_training_form.mentor_training,
+    ),
+  ) do |f| %>
+    <%= f.govuk_error_summary %>
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <span class="govuk-caption-l"><%= t(".add_claim", school: @school.name) %></span>
+        <%= f.govuk_radio_buttons_fieldset(
+          :hours_completed,
+          legend: {
+            size: "l",
+            text: t(".hours_of_training", mentor: mentor_training_form.mentor_full_name),
+          },
+        ) do %>
+          <%= f.govuk_radio_button :hours_completed, 20, label: { text: t(".20_hours"), link_errors: true } %>
+          <%= f.govuk_radio_button :hours_completed, 6, label: { text: t(".6_hours") } %>
+
+          <%= f.govuk_radio_divider %>
+
+          <%= f.govuk_radio_button(
+            :hours_completed,
+            "on",
+            checked: mentor_training_form.custom_hours_checked?,
+            label: { text: t(".other_amount") },
+          ) do %>
+            <h5 class="govuk-heading-s govuk-!-margin-bottom-1"><%= t(".number_of_hours") %></h5>
+            <%= f.govuk_number_field(
+              :custom_hours_completed,
```

Comment:
Why does this need to be `custom_hours_completed`?

Reply:
I think we need to distinguish between the field where the user inputs his custom hours and the preselected hours Don't think rails will magically know, unless I missed something. This is a dirty demo with all fields named `hours_completed`. The error is not targeting the field corectly and when you come and update the form it will not work as you expected.


https://github.com/DFE-Digital/itt-mentor-services/assets/11318084/63028c54-1026-4e21-b8e2-ccedf2afecfa

Reply:
Simon has given the 👌 for us to go with the singular `hours_completed` attribute. https://ukgovernmentdfe.slack.com/archives/C0657JE64HX/p1709291417171859?thread_ts=1709291213.242919&cid=C0657JE64HX

Reply:
Base data-wise, this distinction isn't preferred, as it adds presentational concerns to the raw data.

If we want to learn more about the user behaviour, we can add in analytics tools.

Reply:
I think we still need the `custom_hours_completed` but it will be just a attribute on the form object not on the mentor training model. This is because we need to target it for errors